src/watchmaker/workers/salt.py

```diff
@@ -20,6 +20,12 @@
 from watchmaker.workers.base import WorkerBase
 
 
+def _clear_copy_dir(src, dst):
```

Comment:
If we're going to move this out of the class, then please put it in the `utils` module.

Also, what do you think about overloading the shutil `copytree` function? i.e....

```
def copytree(src, dst, force=False, **kwargs):
    if os.path.exists(dst) and force:
        shutil.rmtree(dst)
    shutil.copytree(src, dst, **kwargs)
```

Reply:
I moved this to utils which made testing much easier. All I changed was switching the order of `if` conditions. I did not override the shutil function but used the same signature. Is that what you wanted?